packages/graphback-runtime-mongodb/tests/MongoDataProviderTest.ts

```diff
@@ -213,4 +231,49 @@ describe('MongoDBDataProvider Basic CRUD', () => {
       expect(todos[t].text).toEqual(`todo${5 - t}`);
     }
   });
+
+  test('createdAt', async () => {
+    context = await createTestingContext(`
+    """
+    @model
+    @versioned
+    """
+    type Note {
+      id: ID!
+      text: String
+    }
+    `);
+    const cDate = new Date(2020, 5, 26, 18, 29, 23);
+    advanceTo(cDate);
+
+    const res = await context.providers.Note.create({ text: 'asdf' });
+    expect(res.createdAt).toEqual(cDate.getTime());
+  })
+
+  test('updatedAt', async () => {
+    context = await createTestingContext(`
+    """
+    @model
+    @versioned
+    """
+    type Note {
+      id: ID!
+      text: String
+    }
+    `);
+    let cDate = new Date(2020, 5, 26, 18, 29, 23);
+    advanceTo(cDate);
+
+    const res = await context.providers.Note.create({ text: 'asdf' });
+    expect(res.updatedAt).toEqual(cDate.getTime());
```

Comment:
Probably worth to add the check that `createdAt` value has not changed

Reply:
Ah, yes, thanks!